web-app/src/components/UserAvatar.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <q-avatar v-bind="avatarProps">
-    <!-- TODO: Replace with user.avatarSrc -->
-    <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
+    <img v-if="userPhoto.startsWith('https')" :src="userPhoto" />
+    <img v-else :src="'data:image/png;base64, ' + userPhoto" />
```

Comment:
I suggest leaving one img tag and move logic to mounted function. There already is some logic anyway